docs/usage/MigPlanPVSelection.md

```diff
@@ -0,0 +1,97 @@
+| copyMethod   | The method which will be used to copy the volume. "snapshot" will use the underlying storage provider's snapshot method, while "filesystem" will use restic (for indirect migration), or rsync (for DVM). Snapshot copy requires the same underlying storage type to be used for both source and destination storage classes, that the storage provider matches the one used for the configured MigStorage, and that it be of a type which supports snapshots -- currently ebs, azure, and gcp. In addition, in most cases, both source and destination clusters must use the same provider region. | "filesystem" | "snapshot", "filesystem" |
+| accessMode   | The access mode to use for the PVC in the destination cluster. If empty, the source cluster PVC's access mode will be used.   | <empty> | "ReadWriteOnce", "ReadWriteMany" |
+| verify       | Whether or not to perform filesystem copy verification after copying volume contents. This is only applicable to filesystem copy. | false | true, false |
+
+Note that `accessMode` is not currently exposed via the web user
+interface, which means that editing the MigPlan yaml is the only way
+to set this value.
+
+The `supported` field will list the available actions and copy methods
+for a given volume. The `selection` value for these fields must be
+found there. For `storageClass`, the selection must be found in the
+list from `migplan.status.destStorageClasses`.
+
+## Other persistent volume fields
+
+The full list of migplan `persistentVolumes` fields is as follows:
+
+| Field             | Description |
+|-------------------|-------------|
+| name              | The name of the source PV |
+| capacity          | The capacity of the source PV |
+| storageClass      | The source PV storageClass |
+| supported         | Supported actions and copy methods for the volume |
+| selection         | User selections for the volume |
+| pvc               | Source PVC metadata (see below for fields) |
+| nfs               | NFS volumesource information copied from PV for NFS volumes |
+| staged            | Internal field used for managing updates |
+| proposedCapacity  | Used for PV resizing for nearly-full volumes |
+| capacityConfirmed | Used for PV resizing for nearly-full volumes |
+
+`pvc` subfields:
+
+| Field        | Description |
+|--------------|-------------|
+| namespace    | The PVC namespace |
+| name         | The PVC name |
+| accessModes  | PVC accessModes |
+| hasReference | Whether or not this PVC is referenced by pods or pod template resources |
+
+## Storageclass status fields
+
+`status.srcStorageClasses` and `status.destStorageClasses` follow the
+same format. The value is a slice of storageclass values. The
+`destStorageClasses` value is the one which is relevant to volume
+selection. `selection.storageClass` must match the `name` field in one
+of the values in `status.destStorageClasses`. The storage class subfields are as follows:
```

Comment:
@sseago  Am I missing something? Where are the `status.destStorageClasses` values? If it is somewhere in this PR, please tag me.

Reply:
@apinnick It's immediately under this comment -- lines 92-97. I just updated the PR yesterday, so it's possible that when you commented you were looking at a previously-loaded browser tab. You may need to reload if that's the case.